en/guide/sdk.md

```diff
@@ -0,0 +1,155 @@
+# DroneCore SDKs

```

Comment:
@julianoes So this is supposed to be a guide on writing extension SDKs, covering just the bits that are relevant to SDK developers only.

Reply:
I would not call it SDK though. I know the Yuneec SDK is called that way but that shouldn't be the default. The goal is to have DroneCore plus extensions from a vendor (without the SDK lock in).